CONTRIBUTING.md

```diff
@@ -129,6 +129,40 @@ Run `rush build -t <packagename>` to build a single project, and all local proje
 
 By default, Rush only displays things written to `STDERR`. If you want to see the full output, pass `--verbose` to any of the build commands. This is also true for script commands (see below).
 
+### Testing
+
+Once the project has successfully built, you can test any of our packages by executing `rushx test` from the root of the package that you want to test. This will automatically run the tests in both NodeJS and in a browser. To target these environments individually, you can run `rushx test:node` and `rushx test:browser`.
+
+By default, these npm scripts run recorded tests. We use a custom recording library called `test-utils-recorder`. You can learn more about our recorder at the [test-utils-recorder's readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/README.md).
+
+#### Recorded tests
+
+To run unit tests, go to the root of the project you want to test and run `rushx unit-test`. These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses. You won't need to have any environment configuration, nor internet access, for these tests to run.
```

Comment:
As mentioned in https://github.com/Azure/azure-sdk-for-js/pull/8398#discussion_r410557291, let's avoid using the term "unit tests" or refering to the npm script `unit-test`

```suggestion
By default, the npm scripts `test`, `test:node` and `test:browser` run tests in playback mode i.e there is no request made to the service. For each http request, the recorded response is used instead which reproduces the service behavior. When running in playback mode, you won't need to have any environment configuration, or internet access.
```

Reply:
Thank you!